### DISCUSSION TITLE
list-disc doesn't work in chrome or safari only in firefox

### DISCUSSION THREAD

**Discussion by kaminski-dev** [tailwindlabs/tailwindcss#12075] [Help] [answered]
My issue is that when using list-[square] or list-disc is working perfectly fine in firefox but not in safari or chrome.
https://play.tailwindcss.com/UWXJLU7kyV

Is something with my code wrong that what needs to be adjusted to work in one of the other browsers?
I also have a postcss.config.js file in my repo with the following content in there
```
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
};

```
Thanks in advance for any help and advice.

**Comment by alex-krasikau** [comment, marked as answer]
Hey @danoshi. The list style works as expected, but these are just implementation details of different browsers. They use different CSS engines that have minor inconsistencies that you can observe here.

This issue is actually related to using `bg-clip-text`, which makes bullets invisible in Chrome and Safari.

To fix this, I would recommend wrapping your text in a span with a gradient background and text clipping, and then styling a bullet separately. I tried to replicate it; you can see the second point's styling here - https://play.tailwindcss.com/Rbu53AELuB.

**Reply by kaminski-dev** [reply]
Thank you very much, this explains why the bullets where not visible. 
I adapted my code and now it works as excpeted, thank you for the help and the example 👍
